data/networks.ts

```diff
@@ -28,12 +32,33 @@ export const l1Networks = {
 export type L1Network = (typeof l1Networks)[keyof typeof l1Networks];
 
 export type EraNetwork = L2Network & {
-  id: 324 | 280 | 270;
+  id: number;
   rpcUrl: string;
-  blockExplorerApi: string;
+  blockExplorerApi?: string;
   faucetUrl?: string;
+  getTokens: () => Token[] | Promise<Token[]>;
 };
 export const eraNetworks: EraNetwork[] = [
+  // See official documentation for in-memory node setup details: https://era.zksync.io/docs/tools/testing/
```

Comment:
maybe do not mention in-memory node, but just testing setup, given it can be the in-memory OR local-setup